### ISSUE TITLE
Item runs should log as INFO as they did in Outset Python

### ISSUE THREAD

**Issue by jazzace** [macadmins/outset#49]
**Describe the bug**
In [Outset Python](https://github.com/chilcote/outset), when an item was run (e.g., a script in login-once), the logs would show a line like:
```2024-01-01 00:00:03 AM - INFO: Processing /usr/local/outset/login-every/login_script.sh```

In this rewrite of Outset, this has been assigned as a DEBUG entry in the logs:
```2024-01-01 00:00:04 DEBUG: Processing script /usr/local/outset/login-every/login_script.sh```

Ideally, the behaviour in this implementation of Outset should match the previous behaviour.

**To Reproduce**
1. Place a script in the login-once or login-every folder within /usr/local/outset/, set to executable by all users.
2. Log in to an account so that Outset runs the item placed in the folder.
3. Examine the logs (/usr/local/outset/logs/outset.log in this version of Outset, /Users/<username>/Library/Logs/outset.log in Outset Python).

**Expected behaviour**
That the output should be the same or similar between this version of Outset and Outset Python.

**Actual Behaviour**
Outset Python shows a line like:
```2024-01-01 00:00:03 AM - INFO: Processing /usr/local/outset/login-every/login_script.sh ```
This version of Outset only shows this information if verbose_logging is turned on in its preferences.

**System Details:**
 - OS: macOS 13.6.4, 17.4RC
 - Architecture: Intel and Apple Silicon M1 respectively
 - Version: 4.1.1.21918

**Comment by jazzace** [comment]
Yes, the logs do fix the issue I raised, but as you suggested, I can only get verbose logs now. It would be good if you reverted to the previous behaviour now that Info items are the same as in Outset Python.

**Comment by bartreardon** [comment]
yeah, I agree - regarding the level of info to debug, is there anything else currently marked as debug that you think should be in info?

**Comment by jazzace** [comment]
I don't think so. I was used to what the Python version gave me.

**Comment by bartreardon** [comment]
this one ok to close? 
If there's any additional stuff you want being logged, LMK and we can chuck it into the 4.2.0 release
